cookbooks/integrating_symfony2_with_behat.rst

```diff
@@ -92,6 +92,29 @@ to change your ``behat.yml`` configuration and tell the extension which services
 This configuration will try to to match the ``$session`` dependency of your ``FeatureContext`` constructor by injecting the ``session`` service into the context.
 Be careful because if such a service does not exist or its name does not match, it will not work and you will end up with a Behat exception.
 
+Accessing Symfony2 parameters in behat.yml
+------------------------------------------
+
+The extension makes it possible to use Symfony's parameters. To make a Symfony parameter available in a context
+you need to change your ``behat.yml`` configuration and tell the extension which parameter to inject:
+
+.. code-block:: yaml
+  default:
+    suites:
+      default:
+          contexts:
+              - FeatureContext:
+                  mySymfonyParam: %%my.symfony_param%%
```

Comment:
@e-weimann I suppose the name was kept to avoid BC breaks, check the [composer.json](https://github.com/Behat/Symfony2Extension/blob/master/composer.json#L22).